README.md

```diff
@@ -2,9 +2,64 @@
 
 MuSIC is a hierarchical map of eukaryotic cell architecture created from integrating immunofluorescence images in the Human Protein Atlas with affinity purification experiments from the BioPlex resource. Integration involves configuring each approach to produce a general measure of protein distance, then calibrating the two measures using machine learning.
 
-Comprehensive information for MuSIC available at **https://nrnb.org/music/**.
+This repository contains the source code and data for reproducing the results in [Qin et al. 2020](https://www.biorxiv.org/cgi/content/short/2020.06.21.163709v1), as well as step-by-step introduction for the MuSIC tool.
+
+The generated MuSIC map and more details are available at [https://nrnb.org/music/](https://nrnb.org/music/).
+
+
+## Installation
+
+Here are the steps for setting up the enviroment for running MuSIC tool:
+
+- Install [Miniconda](https://docs.conda.io/en/latest/miniconda.html), [Anaconda](https://www.anaconda.com/products/individual#Downloads) or other virtual environment manager.
+
+- For Ubuntu (or similar Linux environments) users, run the following command to install the required libraries:
+```bash
+sudo apt install build-essential python-dev libxml2 libxml2-dev zlib1g-dev libigraph0-dev libmpc-dev
+```
+- First create a Python virtual environment, for example, in Conda this would be done with:
+```
+conda create -n music python=3.6.2
+```
+- Then, we need to install the Data-Driven Ontology Toolkit (DDOT) by following the instructions at: https://github.com/michaelkyu/ddot#install-the-ddot-python-package
+
+- Finally, we can clone the MuSIC git repository and install the required Python packages:
+```
+git clone https://github.com/idekerlab/MuSIC
+cd MuSIC
+source activate music # activate the virtual environment
+pip install -r ./requirements.txt
+```
+
+This code was tested on an Ubuntu 18.04 system.
```

Comment:
@y8qin What is the OS you tested? We can also add a list of tested OSs.

@cwinsnes Did you used Ubuntu 18.04?

Reply:
I used Ubuntu 20.04 for my tests.